lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -26,7 +26,7 @@ function dosomething_mbp_install() {
   dosomething_mbp_install_productions();
 
   // Disable Drupal core user event email messages
-  dosomething_mbp_update_7006();
+  dosomething_mbp_update_7006(NULL);
```

Comment:
You better just remove the `&$sandbox` argument in `dosomething_mbp_update_7006` definition.